### ISSUE TITLE
Pre-rendered articles using R.rsp::asis don't display properly

### ISSUE THREAD

**Issue by dmphillippo** [r-universe-org/help#143]
In my package I have a number of vignettes that are pre-rendered due to longer run times; the rendered html files are then included using `R.rsp::asis`.

These all show up correctly in my universe, however the page cannot be scrolled - see [this article](https://dmphillippo.r-universe.dev/ui#view:multinma/example_atrial_fibrillation.html) for example, in comparison to the same [vignette](https://dmphillippo.github.io/multinma/articles/example_atrial_fibrillation.html) on the pkgdown site.

It seems to be something to do with the height of the iframe that contains the rendered article, but what or why is beyond me!

**Comment by jeroen** [comment]
Yes it works correctly from the direct url (that is also linked at the top of the dashboard): https://dmphillippo.r-universe.dev/articles/multinma/example_atrial_fibrillation.html


<img width="1081" alt="Screenshot 2022-04-13 at 5 22 02 PM" src="https://user-images.githubusercontent.com/216319/163214825-66fafaa4-b1a0-4220-a7a1-f431a70e0dee.png">

The problem is that we inject [iframeresizer code](https://r-universe.dev/static/article.js) into the vignette html when we generate the articles, however your asis html files don't have this right now. Also the markup is ugly of course.

How is it possible that pkgdown is able to generate a pretty custom formatted article from your as-is html file?

**Comment by dmphillippo** [comment]
Ah that makes sense, thanks!

Well, I actually have pkgdown set up to rebuild the vignettes from the Rmds rather than using the asis html. I'm not sure the latter would be possible, and probably not pretty! The pre-compilation is only really needed to keep build time down on CRAN.

Is it possible for r-universe to inject the script to the asis html after the fact?

**Comment by dmphillippo** [comment]
Thanks Jeroen. I did look at the Rmd to md option at the time, but I can't remember now why I went down the html asis route instead. I agree that your way does make more sense!
